packages/ui/package.json

```diff
@@ -24,7 +24,9 @@
   "devDependencies": {
     "@types/qrcode": "1.5.0",
     "eslint-plugin-lit": "1.6.1",
-    "eslint-plugin-wc": "1.3.2"
+    "eslint-plugin-wc": "1.3.2",
+    "rollup-plugin-scss": "^3.0.0",
```

Comment:
rollup plugin for scss is also already defined in root package.json, lets keep all rollup lugins there